slm_porosity/problem.inp

```diff
@@ -160,7 +163,9 @@ lfrac_scale = 10                            # scale factor for mesh adaptation
 power_factor_2d = 0.3                       # laser power factor for the 2D problem
 tol_newton = 1e-6                           # tolerance for root finding algorithm
 max_iter_newton = 10                        # maximum number of iterations for root finding algorithm
-check_resolution = T                        # if true, abort program in case of the unsifficient mesh resolution
+check_resolution = T                        # if true, abort program in case of the insufficient mesh resolution
 thickness_points = 8                        # minimum number of points inside the solid--liquid interface thickness
 j_mx_porosity = 7                           # J_MX for porosity
 cfl_fusion_factor = 2                       # the time step will be reduced this number of times
+laser_as_bc = T                             # T - laser as BC, F - internal heating source
+half_domain = T                             # T - symmetry across laser movement axis, F - no symmetry
```

Comment:
Let it be `# if true, solve the problem only in a half domain for the 3D case`

Reply:
It is wrong. Let's say I put `initial_laser_position` (0,-2,0) than laser would move across one half of the domain. But we changed BC and there is nothing about it. This flag do not divide domain by half. It changes BC rules for creating symmetry

Reply:
Still the laser beam moves along the x axis; therefore, your case means that we have two laser beams moving in parallel.